Initial release

Elektroarzt opened the pull request.

Elektroarzt committed:
Initial release

jnsbyr commented:
Your PCB design is really nice. It needs a slightly larger case but is a lot more uncluttered than my prototype. Will update the Readme during this week merge.

PiperNigrum commented:
Hi Elektroarzt,

i would like to use your pcb but have no idea which steps I have to do.
Did you ordered your pcb an soldered all parts by your own?
Is it possible to "order" a piece by you which is ready to use?

I look forward to your response.

Thanks.

Best regards
Tobias

Elektroarzt commented:
Hi Tobias,

thanks for your interest in the PCB. I designed, ordered and soldered it on my own as I'm Electronics Engineer (thanks to Jens for the fabulous schematic :).
Seems you have no hardware background, which probably makes it way of difficult to integrate the PCB in your pool in case of issues and debugging even if you have a proper soldered PCB. Also be aware the pool runs on mains with 230V, so be sure to know what you are doing.
Nevertheless, if you want to go on with all this said, I have a soldered PCB for you. For details you can reach me by mail at elektroarzt@digital-filestore.de.
Please keep in mind, you need some piece of software behind your MQTT broker and you also have to configure the D1 mini according to your infrastructure to get the system up and running.

Best regards,
Thomas

jnsbyr committed:
README updated

- README updated to show PCB
- PCB photos renamed so they can be referenced from README
